### PR TITLE
fix versions

### PR DESCRIPTION
**Pull request by kaladivo**

### CODE REVIEW

**Comment by kaladivo** [comment]
Current dependencies on/for this PR:
* **#822** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/822?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#824** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/824?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#796** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/796?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#821** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/821?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#814** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/814?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#809** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/809?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#808** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/808?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#807** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/807?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#806** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/806?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#804** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/804?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#798** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/798?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#797** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/797?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#795** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/795?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#794** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/794?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/vexl-it/vexl/814?utm_source=stack-comment).

File: apps/mobile/src/state/chat/atoms/checkAndReportCurrentVersionToChatsActionAtom.ts [hunk 1]
```diff
@@ -49,8 +53,11 @@ const sendUpdateNoticeMessageActionAtom = atom(
     const messageToSend = createUpdateNoticeChatMessage({
       version,
       senderPublicKey: chat.chat.inbox.privateKey.publicKeyPemBase64,
+      chat: chat.chat,
     })
 
+    console.log(`Sending update: ${JSON.stringify(messageToSend, null, 2)}`)
```
Comment (SamTremko) [inline apps/mobile/src/state/chat/atoms/checkAndReportCurrentVersionToChatsActionAtom.ts:59]: pls remove

File: apps/mobile/src/state/chat/atoms/fetchNewMessagesActionAtom.ts [hunk 2]
```diff
@@ -33,10 +33,44 @@ import {
...
+      inbox,
+    }: {newMessages: readonly ChatMessageWithState[]; inbox: InboxInState}
+  ) => {
+    const messagesToRespondWithCurrentVersion = newMessages.filter(
+      (one) =>
+        one.message.messageType === 'VERSION_UPDATE' &&
+        one.message.lastReceivedVersion !== version
+    )
+
+    console.log(`New messages: ${JSON.stringify(newMessages, null, 2)}`)
```
Comment (SamTremko) [inline apps/mobile/src/state/chat/atoms/fetchNewMessagesActionAtom.ts:60]: pls remove

File: packages/resources-utils/src/chat/utils/parseChatMessage.ts [hunk 3]
```diff
@@ -153,8 +154,11 @@ export function parseChatMessage({
   ErrorParsingChatMessage | ErrorChatMessageRequiresNewerVersion,
   ChatMessage
 > {
-  return (jsonString) =>
-    pipe(
+  return (jsonString) => {
+    console.log(
```
Comment (SamTremko) [inline packages/resources-utils/src/chat/utils/parseChatMessage.ts:158]: pls remove

File: packages/resources-utils/src/chat/utils/serializeChatMessage.ts [hunk 4]
```diff
@@ -46,6 +46,7 @@ function deanonymizedUserToPayload(
 export default function serializeChatMessage(
   message: ChatMessage
 ): E.Either<JsonStringifyError | ZodParseError<ChatMessagePayload>, string> {
+  console.log(`Serializing chat message: ${JSON.stringify(message)}`)
```
Comment (SamTremko) [inline packages/resources-utils/src/chat/utils/serializeChatMessage.ts:49]: pls remove

**Review by SamTremko — APPROVED** [review]

**Comment by kaladivo** [comment]
### Merge activity

* **Feb 26, 5:08 AM EST**: [Graphite](https://app.graphite.dev/github/pr/vexl-it/vexl/814) rebased this pull request after merging its parent, because this pull request is set to merge when ready.
* **Feb 26, 5:13 AM EST**: @kaladivo merged this pull request with [Graphite](https://app.graphite.dev/github/pr/vexl-it/vexl/814).

**Pull request merged**